tests/nnfw_api/src/GenModelTests.cc

```diff
@@ -193,3 +193,99 @@ TEST_F(GenModelTest, OneTensor_ConstAndThreeOutputs)
 
   SUCCEED();
 }
+
+TEST_F(GenModelTest, Reshape_with_shape_param_as_const)
```

Comment:
What about removing `param_` in the test name? "shape param" sounds like it is an operation's parameter which is not. a tensor, to me.

Reply:
What I meant was param of Reshape op.

in https://github.com/Samsung/ONE/pull/4738/files#diff-4ee790f1f7d02d361d6728f1b1c16729ff0f9afffbed34de279a2a3a3edb482bR211, 

I meant the following **bold** param.
cgen.addOperatorReshape({{input, new_shape}, {out}}, **&new_shape_data**); 

Since the main change of this PR is about handling param as an option (if omitted, param shape is `nullptr`)
I intended to add two test cases: one with param, another without param.

Reply:
Oh thank you for explanation! I forgot Reshape has "new shape" as a param as well as an input.